test/common/upstream/upstream_impl_test.cc

```diff
@@ -278,7 +278,7 @@ TEST_F(StrictDnsClusterImplTest, ZeroHostsHealthChecker) {
   EXPECT_EQ(0UL, cluster.prioritySet().hostSetsPerPriority()[0]->healthyHosts().size());
 }
 
-TEST_F(StrictDnsClusterImplTest, Basic) {
+TEST_F(StrictDnsClusterImplTest, DEPRECATED_FEATURE_TEST(Basic)) {
```

Comment:
Changes in this file seem unrelated. Revert file please.

Reply:
Reverted.